### ISSUE TITLE
Connect user dashboard to backend

### ISSUE THREAD

**Issue by ChantelleKerr** [codersforcauses/csf#112]
## Prerequisite
*Do not start this issue until this has been merged into main*
#87 Implement user in pinia store 

## Basic Information
- Display user name in "Welcome back section"
- Display correct icon for travel method - 'RUNNING', 'WHEELING', 'WALKING'

**Comment by ChantelleKerr** [comment]
issue #87 has been merged so this can be started
